packages/language-support/src/antlr-grammar/CypherPreParser.g4

```diff
@@ -8,4 +8,4 @@ preparsedStatement:
    preparserOption? statement;
 
 preparserOption:
-   EXPLAIN | PROFILE;
\ No newline at end of file
+   EXPLAIN | PROFILE | CYPHER (UNSIGNED_DECIMAL_INTEGER | IDENTIFIER EQ (IDENTIFIER | numberLiteral))*;
```

Comment:
It would be cleaner to extract the Cypher part to another rule?

```
preparserOption:
  EXPLAIN | PROFILE | cypherOptions

cypherOptions:
   CYPHER (cypherVersion | cypherOption)*
```